src/main/kotlin/no/nav/tilleggsstonader/kontrakter/søknad/FeltTyper.kt

```diff
@@ -3,15 +3,9 @@ package no.nav.tilleggsstonader.kontrakter.søknad
 import java.time.LocalDate
 import java.time.YearMonth
 
-data class BooleanFelt(
-    val label: String,
-    val verdi: Boolean,
-)
-
 data class TekstFelt(
     val label: String,
     val verdi: String,
-    val svarTekst: String? = null,
```

Comment:
`svarTekst` finnes fortsatt på `EnumFelt`, har vi noe bedre navn på den? 
Tenker også jeg skal dokumentere hva som menes med den.

Tanken er at den skal inneholde det tekstlige som vises for brukeren

Reply:
valgtSvar 🤔